yggdrasil.go

```diff
@@ -150,6 +144,11 @@ func main() {
 		if err := hjson.Unmarshal(config, &dat); err != nil {
 			panic(err)
 		}
+		confJson, err := json.Marshal(dat)
```

Comment:
Appears that this does successfully fix the problem where setting `"MulticastInterfaces"` to `[]` didn't work.